packages/rocketchat-emoji-emojione/package.js

```diff
@@ -13,9 +13,10 @@ Package.onUse(function(api) {
 		'rocketchat:lib'
 	]);
 
-	api.addFiles('emojiPicker.js', 'client');
+	api.addFiles('emojiPicker.js', ['client', 'server']);
```

Comment:
Use `api.addFiles('emojiPicker.js');` instead of `api.addFiles('emojiPicker.js', ['client', 'server']);`